### ISSUE TITLE
Feature Request: export and migration to private kutt.it

### ISSUE THREAD

**Issue by xinbenlv** [thedevs-network/kutt#106]
## Features
1. export links created under an account to a portable structured format such as JSON
2. import the data for a self-hosted kutt instance.
3. [optional] redirect kutt.it link to the ported new instance's respective link.
## Use Case
I run communities and constantly need to set up new communities. For one-off shortlink I can use kutt.it, once the community take off, I'd like to be able to self-host the links and port all link created over. (and update the domain of reference of course)

**Comment by poeti8** [comment]
This is a useful one. But I'm working on another The Devs project and can't start working on this right now. The closes time that I can get back to Kutt is within the next month probably.

**Comment by brownsugar** [comment]
Any update for this? I want to import some links with custom path and clicks from another service, this would be helpful.

**Comment by trgwii** [comment]
@brownsugar I think your usecase is more ambitious than what @xinbenlv is suggesting, he only means exporting links from a Kutt instance into another Kutt instance, not a different URL-related service.

**Comment by brownsugar** [comment]
Yep, but I think if this feature is added, I could list my custom data with certain format like json to import.

**Comment by Likenttt** [comment]
I find it not easy to migrate postgres db to a new one cuz of trigger constraints even though I disable the function with `--disable-triggers`
